<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcceptReques</class>
 <widget class="QWidget" name="AcceptReques">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 14pt &quot;Candara&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="acceptStkW">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="question">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
                 background-color: white;
                 color: black;
                 border: 1px solid rgb(110, 110, 110);
              border-radius: 10px;
                 padding: 3px 7px;
             }

             QLineEdit:focus {
                 border: 2px solid rgb(90, 90, 90);
            }QPushButton
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 20px;
box-shadow: 0 0 20px 10px rgb(240, 158, 255);
width: 90px;
  height: 90px;
}
QPushButton:hover {
               border: 2px solid rgb(200, 200, 200);
            }
QPushButton:pressed {
             background-color: rgb(255, 255, 255);
	        color: rgb(0, 0, 0);
            }</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="3">
        <widget class="QPushButton" name="refusePb">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="picture.qrc">
           <normaloff>:/photo/refuse.jpg</normaloff>:/photo/refuse.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="accepPB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="picture.qrc">
           <normaloff>:/photo/acceptJpg.jpg</normaloff>:/photo/acceptJpg.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Do you want to accept this application?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="addNewCustomer">
      <property name="styleSheet">
       <string notr="true">QWidget
{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
	font: 9pt &quot;Candara&quot;;
}
QTableView {
	font: 9pt &quot;Century Gothic&quot;;
    border: 3px solid black;
    border-radius: 10px;
    background-color: #f0f0f0;  /* Фон заокругленого віджета */
    selection-background-color: #a6a6a6;  /* Колір виділення */
}
/* Імена стовбців */
QHeaderView::section {
    background-color: #404040;  /* Колір фону */
    color: white;  /* Колір тексту */
    padding: 4px;
    border: 1px solid #303030;  /* Границі */
}

/* Номерація рядків */
QTableView QTableCornerButton::section {
    background-color: #404040;  /* Колір фону */
    color: white;  /* Колір тексту */
    border: 1px solid #303030;  /* Границі */
}


QTableView::item:hover {
    border: 2px solid #505050;  /* Колір границі при наведенні */
}

QTableView::item:selected {
    border: 2px solid #303030;  /* Колір границі при виділенні */
}
QLineEdit {
font: 12pt &quot;Candara&quot;;
                 background-color: white;
                 color: black;
                 border: 1px solid rgb(110, 110, 110);
              border-radius: 10px;
                 padding: 3px 7px;
             }

             QLineEdit:focus {
                 border: 2px solid rgb(90, 90, 90);
            }
QPushButton
{
	font: 10pt &quot;Candara&quot;;
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 20px;
box-shadow: 0 0 20px 10px rgb(240, 158, 255);
width: 50px;
  height: 40px;
}
QPushButton:hover {
               border: 2px solid rgb(200, 200, 200);
            }
QPushButton:pressed {
             background-color: rgb(255, 255, 255);
	        color: rgb(0, 0, 0);
            }
QLabel{font: 12pt &quot;Candara&quot;;}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="4">
        <widget class="QLabel" name="masterLb">
         <property name="text">
          <string>Master:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5" colspan="3">
        <widget class="QLineEdit" name="priceLE"/>
       </item>
       <item row="1" column="1" colspan="5">
        <widget class="QTableView" name="customerTv"/>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QLineEdit" name="numberLE"/>
       </item>
       <item row="4" column="0" rowspan="3">
        <widget class="QLabel" name="numberLb">
         <property name="text">
          <string>Number:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="searchLbl">
         <property name="styleSheet">
          <string notr="true">QLabel {
    border: 1px solid #000000; /* Задайте границю */
    border-top-left-radius: 15px; /* Заокруглення тільки з одного боку */
    border-bottom-left-radius: 15px;
    border-top-right-radius: 0; /* Встановіть інші кути без заокруглення */
    border-bottom-right-radius: 0;
    background-color: rgb(255, 255, 255); /* Колір фону */
    color: #000000; /* Колір тексту */
    padding: 5px; /* Відступи */
    width: 35px;
    height: 30px;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="picture.qrc">:/photo/search.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="nameLb">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="surenameLb">
         <property name="text">
          <string>Surename:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLineEdit" name="nameLE"/>
       </item>
       <item row="3" column="4" rowspan="2">
        <widget class="QLabel" name="bodyLb">
         <property name="text">
          <string>Body:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5" colspan="3">
        <widget class="QComboBox" name="bodyCb"/>
       </item>
       <item row="6" column="5" colspan="3">
        <widget class="QComboBox" name="mastersCb"/>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="priceLb">
         <property name="text">
          <string>Price:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="3">
        <widget class="QLineEdit" name="searchLE">
         <property name="styleSheet">
          <string notr="true"> border: 1px solid #000000; /* Задайте границю */
    border-top-left-radius: 0px; /* Заокруглення тільки з одного боку */
    border-bottom-left-radius: 0;
    border-top-right-radius: 15px; /* Встановіть інші кути без заокруглення */
    border-bottom-right-radius:15px;
    background-color: rgb(255, 255, 255); /* Колір фону */
    color: #000000; /* Колір тексту */
    padding: 5px; /* Відступи */
    width: 665px;
    height: 30px;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLineEdit" name="surenameLE"/>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QPushButton" name="bodyPb">
         <property name="text">
          <string>Body</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="wantArea">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="5" colspan="2">
        <widget class="QStackedWidget" name="buttonStkw">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="addPbPage">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QPushButton" name="addPb">
             <property name="text">
              <string>ADD</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="updatePbPage">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QPushButton" name="updatePb">
             <property name="text">
              <string>UPDATE</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
